src/codeu/chat/util/Time.java

```diff
@@ -89,18 +86,23 @@ public boolean equals(Object other) {
   public int hashCode() {
     return new Long(inMs()).hashCode();
   }
-  
+
   public static Time parse(String s) {
-	try {
-	  return new Time(formatter.parse(s).getTime());
-	} catch (ParseException e) {
-	  e.printStackTrace();
-	}
-	  return Time.now();
+    try {
+      return new Time(formatter.parse(s).getTime());
+    } catch (ParseException e) {
+      e.printStackTrace();
+    }
+    return Time.now();
   }
 
   // Difference of end and start times.
+  // DEPRECATED(Adam): Use Duration.java instead.
```

Comment:
Use `@Deprecated` annotation. Maybe turn this comment into a TODO to migrate existing code.